CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.0.1 (2020-08-05)
+
+- Fix the progress indicator component to not error when the progress indicator and its external navigation buttons are hidden at the same time. [#40](https://github.com/blackbaud/skyux-progress-indicator/pull/46)
```

Comment:
Add an -ed to Fix. Also, "to not error" reads a little awkward. Would "to prevent errors" work instead or would that change the meaning?

Reply:
Done